.github/workflows/update_disabled_tests.yml

```diff
@@ -41,3 +41,9 @@ jobs:
           user_email: "test-infra@pytorch.org"
           user_name: "Pytorch Test Infra"
           commit_message: "Updating condensed disabled tests stats"
+
+      - name: Upload file to s3
+        run: python3 .github/scripts/upload_to_s3.py "disabled-tests-condensed.json"
```

Comment:
Hmm, I don't think you need to have a Python script for such simple operation, you could try to install `awscli` with pip and just run `aws s3 cp debug.txt s3://ossci-metrics/debug.txt`